kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/MetricsIT.java

```diff
@@ -152,6 +165,59 @@ void prometheusMetricsWithCommonTags(KafkaCluster cluster) {
         }
     }
 
+    @Test
+    void testKroxyliciousInboundDownstreamMessages(KafkaCluster cluster, Topic topic) {
+
+        NamedFilterDefinition namedFilterDefinition = new NamedFilterDefinitionBuilder("filter-1",
+                ProduceRequestTransformation.class.getName())
+                .withConfig("transformation", TestEncoderFactory.class.getName()).build();
+
+        var config = proxy(cluster)
+                .addToFilterDefinitions(namedFilterDefinition)
+                .addToDefaultFilters(namedFilterDefinition.name())
+                .withNewManagement()
+                .withNewEndpoints()
+                .withNewPrometheus()
+                .endPrometheus()
+                .endEndpoints()
+                .endManagement();
+
+
+        try (var tester = kroxyliciousTester(config);
+                var ahc = tester.getManagementClient();
+                var producer = tester.producer(Map.of(CLIENT_ID_CONFIG, "shouldModifyProduceMessage", DELIVERY_TIMEOUT_MS_CONFIG, 3_600_000));
+                var consumer = tester
+                        .consumer(Serdes.String(), Serdes.ByteArray(), Map.of(GROUP_ID_CONFIG, "my-group-id", AUTO_OFFSET_RESET_CONFIG, "earliest"));
+        ) {
+
+            var metricList = ahc.scrapeMetrics();
+
+            var inboundDownstreamMessagesMetricsValue = metricList.stream().filter(simpleMetric -> simpleMetric.name()
+                    .equals("kroxylicious_inbound_downstream_messages_total")).findFirst().get().value();
+            var inboundDownstreamDecodedMessagesMetricsValue = metricList.stream().filter(simpleMetric -> simpleMetric.name()
+                    .equals("kroxylicious_inbound_downstream_decoded_messages_total")).findFirst().get().value();
+
+            consumer.subscribe(Set.of(topic.name()));
+
+            producer.send(new ProducerRecord<>(topic.name(), "my-key", "hello-world")).get();
+
+            //updated metrics after some message were produced
+            metricList = ahc.scrapeMetrics();
+
+            var updatedInboundDownstreamMessagesMetricsValue = metricList.stream().filter(simpleMetric -> simpleMetric.name()
+                    .equals("kroxylicious_inbound_downstream_messages_total")).findFirst().get().value();
+            var updatedInboundDownstreamDecodedMessagesMetricsValue = metricList.stream().filter(simpleMetric -> simpleMetric.name()
+                    .equals("kroxylicious_inbound_downstream_decoded_messages_total")).findFirst().get().value();
+
+            Assertions.assertTrue(inboundDownstreamMessagesMetricsValue < updatedInboundDownstreamMessagesMetricsValue);
+            Assertions.assertTrue(inboundDownstreamDecodedMessagesMetricsValue < updatedInboundDownstreamDecodedMessagesMetricsValue);
+
+        }
+        catch (ExecutionException | InterruptedException e) {
```

Comment:
The catch block in junit test is just pointless clutter.  Change the test method to `throws Exception`